src/marqo/s2_inference/s2_inference.py

```diff
@@ -412,7 +413,9 @@ def _float_tensor_to_list(output: FloatTensor, device: str = get_default_device(
     Returns:
         List[List[float]]: _description_
     """
-
+    
+    # Hardcoded since get_default_device() used to return "cpu" anyway
+    device = "cpu"
     return output.detach().to(device).tolist()
```

Comment:
this one must be hardcoded as "CPU", as we can only convert a tensor in "CPU" to list.

Reply:
done!